src/main/java/org/jabref/preferences/PreferencesService.java

```diff
@@ -96,5 +98,9 @@
 
     void saveCustomEntryTypes();
 
-    public Boolean getAllowIntegerEdition();
+    Boolean getAllowIntegerEdition();
```

Comment:
Try to use the lowercase boolean, the real "type" and not the wrapper object to avoid auto-boxing of types